src/components/SystemsTable/ConnectionStatusCol.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
+import {
+  ConnectedIcon,
+  DisconnectedIcon,
+  TimesIcon,
+} from '@patternfly/react-icons';
+import { Flex, Popover, Title } from '@patternfly/react-core';
+
+const ConnectionStatusColumn = ({ connection_status, executor_type }) => {
+  console.log(connection_status, 'status here + ', executor_type, 'execture');
+  return (
+    <Fragment>
+      {connection_status === 'connected' && (
+        <span>
+          <ConnectedIcon className="pf-u-mr-xs" /> Connected
+        </span>
+      )}
+      {connection_status === 'disconnected' && executor_type === 'RHC' && (
```

Comment:
Would break down these conditional component renders help to make the code even better readable? What I am imagining is that you would have here if/else conditions or somehow switch case and you would extract the conditionally rendered JSX to separate components

Reply:
i broke it down into if/else to make it more readable